cmd/cmd_test.go

```diff
@@ -185,6 +185,80 @@ sync-ddl = true
 	})
 }
 
+func (s *decodeFileSuite) TestAndWriteExampleServerTOML(c *check.C) {
+	defer testleak.AfterTest(c)()
+	content := `
```

Comment:
Can we use the sample config file here?

Reply:
which block in the example config file do you want to remove?

Reply:
What I mean is to reuse the same sample config, so once a new config is added, we don't need to modify multiple places

Reply:
ok

Reply:
updated